scripts/zones/Uleguerand_Range/npcs/Buffalostalker_Dodzbraz.lua

```diff
@@ -1,22 +1,30 @@
 -----------------------------------
 -- Area: Uleguerand Range
---  NPC: Buffalostalker Dodzbraz
+-- NPC: Buffalostalker Dodzbraz
 -- Type: Quest NPC
 -- !pos -380.171 -24.89 -180.797 5
+-- !additem 1667 (Cluster Core)
 -----------------------------------
 local entity = {}
 
 entity.onTrade = function(player, npc, trade)
+    if npcUtil.tradeHas(trade, { { 1667, 2 } }) then -- Cluster Core x2
+        player:startEvent(8, 1667)
+    end
 end
 
 entity.onTrigger = function(player, npc)
-    player:startEvent(6)
+    player:startEvent(6, 1667)
 end
 
 entity.onEventUpdate = function(player, csid, option)
 end
 
 entity.onEventFinish = function(player, csid, option)
+    if csid == 8 then
+        npcUtil.giveItem(player, 5267)
```

Comment:
https://ffxiclopedia.fandom.com/wiki/Bombs_Away!

```
... to complete the quest.
``` 

Does this leave a completed marker in the quest log?

Reply:
As far as I know, there's no fame involved and since there's no quest in the quest folder to attach/check it to, I feel that isn't not exactly a "quest" other than a simple trade/get that's repeatable. I'm just doing a quick fix so that players can get access to the Shumeyo Salt since there's no other way. I will write up the appropriate structure at another time since I'll have to learn how that structure is and works.

Reply:
What I mean is: on retail, is there an entry in the quest log when you complete this, or is it a hidden quest?

Reply:
There's an entry here: https://github.com/LandSandBoat/server/blob/base/scripts/globals/quests.lua#L573

Which maybe means it has an entry when you complete it, so you'll have to possibly check against that flag, and set it the first time you complete it. You'll also have to find out is there a point of "I've been told about this quest and it appears in my quest log, but I haven't completed it yet". Probably there isn't, but it might be.

There are caps in the caps discord:
https://discord.com/channels/443544205206355968/446470447240970242/643617717986787358

Reply:
I think this might be a hidden quest. The only reason why it's really known is because it's mentioned in the walkthrough. I don't think anyone during the quest mentions an orc in ule-range that knows a weakness to the Snoll Tzar bomb. 

So it DOES have something to check to. I'll have to test this out. I already have lines that I had previously taken out that DID test whether it could be flagged or not but nothing worked as I had nothing to check to. So 96? 

So i had this in the line of the OnTrade part

`if player:getQuestStatus(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.BOMBS_AWAY) ~= QUEST_AVAILABLE
`
this in OnTrigger

`player:getQuestStatus(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.BOMBS_AWAY) == QUEST_AVAILABLE
`
and this in OnEventFinish

`player:completeQuest(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.BOMBS_AWAY)`


I assumed these would work but I didn't get flagged for any so I just took it out and focused on making it just work.